doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+    \draw[dashed](mA.south east)--(mC.south east);
+
+    \node [above left] at (mC.north) {Setting};
+    \node [right] at (mC.east) {Need};
+    \node [left] at (mC.west) {$Y_n$};
+
+    \matrix (mD) [draw,matrix of math nodes] at ($(mA.south east)+(0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_k} & t_{s_2,n_1,ay_k} & \dots & t_{s_i,n_1,ay_k} \\
+      t_{s_1,n_2,ay_k} & t_{s_2,n_2,ay_k} & \dots & t_{s_i,n_2,ay_k} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_k} & t_{s_2,n_n,ay_k} & \dots & t_{s_i,n_j,ay_k} \\
+    };
+
+    \matrix (mE) [draw,matrix of math nodes] at ($(mB.south east)+(0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_2} & t_{s_2,n_1,ay_2} & \dots & t_{s_i,n_1,ay_2} \\
+      t_{s_1,n_2,ay_2} & t_{s_2,n_2,ay_2} & \dots & t_{s_i,n_2,ay_2} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_2} & t_{s_2,n_n,ay_2} & \dots & t_{s_i,n_j,ay_2} \\
+    };
+
+    \matrix (mF) [draw,matrix of math nodes] at ($(mC.south east) + (0,-3.5)$)
+    {
+      t_{s_1,n_1,ay_1} & t_{s_2,n_1,ay_1} & \dots & t_{s_i,n_1,ay_1} \\
+      t_{s_1,n_2,ay_1} & t_{s_2,n_2,ay_1} & \dots & t_{s_i,n_2,ay_1} \\
+      \vdots & \vdots & \ddots & \vdots \\
+      t_{s_1,n_j,ay_1} & t_{s_2,n_n,ay_1} & \dots & t_{s_i,n_j,ay_1} \\
+    };
+
+    \draw[dashed](mD.north east)--(mF.north east);
+    \draw[dashed](mD.north west)--(mF.north west);
+    \draw[dashed](mD.south east)--(mF.south east);
+
+    \draw[dotted,->](mC.south)--(mE.north);
+    \draw[dotted,->](mB.south)--(mD.north);
+
+    \node [left] at (mF.west) {$Y_{n+1}$};
+  \end{tikzpicture}
+  \caption{State Transitions in the Send Markov Chain\label{state}
+  }
+\end{figure}
+
+\subsection{Markov Chain}
+
+This section and the next are a detour into how a the Send model would
+look if it were implemented using just Markov Chains.  If we
+understand this we understand why we can see the Send Model is''
+`Markov like`''.
+
+The initial state of our Markov Chain, $Y_0$ is constructed from the
+historic data provided by the client (we simply count the number of
+pupils that appear e.g for a single state index, $(s_1,n_1,ay_1)$, how
+many rows in our input data match?
+
+In Fig.\ref{state} $Y_{n+1}$ is the next state of our Markov Chain.
+
+How we transition from $Y_{n}$ to $Y_{n+1}$ is the subject of the next
+section.  However, it's worth noting now that the dotted arrows on the
+diagram represent how the $t$'s of one academic year transition into
+the next academic year, in the following calendar year.
+
+The complete state of the model is represented as $S = \{Y_0, \dots,
+Y_n\}$ where $n$ is the number of years we run the simulation for.
+
+\subsection{Markov Transition Matrix}
+
+This section can be skipped - it explores what Markov Transition
+Matrix would look like if the model used it.
+
+In mathematics a Markov Transition Matrix is formally a matrix
+containing the probability of transitioning between Markov
```

Comment:
what is a Markov state as opposed to the other kinds of state we've been talking about?